#goal definition
---
# result
bool sample_present
---
# feedback
